docs/environment-rtd.yaml

```diff
@@ -3,31 +3,31 @@ channels:
 - main
 dependencies:
 - beautifulsoup4=4.9.3
-- black=20.8b1
+- black=21.4b2
 - click=7.1.2
 - codecov=2.1.11
 - dill=0.3.3
-- gcsfs=0.8.0
+- gcsfs=2021.4.0
```

Comment:
This break gcsfs so badly that it just stops writing any data.

Reply:
That's weird. Their [changelog](https://gcsfs.readthedocs.io/en/latest/changelog.html#version-2021-04-0) doesn't seem to mention any breaking changes. What kind of errors are you seeing?

Reply:
Invalid request.  The number of bytes uploaded is required to be equal or greater than 262144, except for the final request (it's recommended to be the exact multiple of 262144).  The received request contained 3524 bytes, which does not meet this requirement., 400

Reply:
> Their changelog doesn't seem to mention any breaking changes

Yeah, I also looked at the comparing it to the 0.8.0 release and almost nothing changed.

Reply:
But the crawl I attempted to run based on the lastest commit in this branch tried to write 11k different files before I realized that nothing was written out.